lib/ember_cli/path_set.rb

```diff
@@ -63,18 +67,19 @@ def build_error_file
     def bower
       @bower ||= begin
         bower_path = app_options.fetch(:bower_path) { which("bower") }
+        bower_executable = Pathname(bower_path.to_s)
```

Comment:
@geoffharcourt I plan to merge https://github.com/thoughtbot/ember-cli-rails/pull/533 ahead of this PR.